docs/newbs_getting_started.md

```diff
@@ -42,12 +42,16 @@ We've tried to make QMK as easy to set up as possible. You only have to prepare
 You will need to install MSYS2, Git, and the QMK CLI.
 
 * Follow the installation instructions on the [MSYS2 homepage](http://www.msys2.org).
-* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal.
+* Close any open MSYS2 terminals and open a new MSYS2 MinGW 64-bit terminal. NOTE: This is **not** the same as the MSYS terminal that opens when installation is completed.
 
-After opening a new MSYS2 MinGW 64-bit terminal run these commands:
+After opening a new MSYS2 MinGW 64-bit terminal, make sure `pacman` is up to date with:
+
+    pacman -Syu
+
+You may be asked to close and reopen the window. Do this and keep running the above command until it says `there is nothing to do`. Then run the following:
 
     pacman -S git python3-pip
-    python3 -m pip install qmk
```

Comment:
This was a deliberate choice to avoid the problem where there are two pythons and one doesn't have pip3. This way we always install for the python3 in the path.

Reply:
At this point, assuming a fresh install of MSYS2, there will only be python 3.7 installed. IMO it's pretty safe.

Reply:
Even still, the recommended style by python is the `python3 -m pip` form as shown here: https://docs.python.org/3/installing/index.html

Reply:
That didn't really explain much, it also says:
> `pip` is the preferred installer program. Starting with Python 3.4, it is included by default with the Python binary installers.

[This issue](https://bugs.python.org/issue22295) is clearer why `python -m pip` is preferred.